app/models/variable_draft.rb

```diff
@@ -8,6 +8,17 @@ class << self
     def forms
       []
     end
+
+    def create_from_variable(variable, user, native_id)
+      draft = VariableDraft.find_or_initialize_by(native_id: native_id)
+      draft.entry_title = (variable['LongName'].blank? ? nil : variable['LongName'])
```

Comment:
if `'LongName'` doesn't exist, the `[]` operator will return `nil`, this ternary operator is unnecessary. (same thing goes for below)